extensions/amp-accordion/0.1/amp-accordion.css

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Non-overridable properties */
+amp-accordion {
+  display: block !important;
+}
+
+/* Make sections non-floatable */
+amp-accordion > section {
+  float: none !important;
+}
+
+/* Hide all children and make them non-floatable */
+amp-accordion > section > *:nth-child(n) {
+  display: none !important;
+  float: none !important;
+}
+
+/*  Display the first 2 elements (heading and content) */
+amp-accordion > section > .amp-accordion-header,
+amp-accordion > section > .amp-accordion-content {
+  display: block !important;
+  overflow: hidden !important; /* clearfix */
+  position: relative !important;
+}
+
+amp-accordion,
+amp-accordion > section,
+amp-accordion > section > .amp-accordion-header,
+amp-accordion > section > .amp-accordion-content {
+  margin: 0;
```

Comment:
Just `.amp-accordion-header, .amp-accordion-content {margin: 0;}`. This will make it easier to override.